cutefetch

```diff
@@ -75,6 +77,14 @@ kittyfetch() {
     echo "             $c4$n$t  $net"
 }
 
```

Comment:
Maybe add a comment like "sysinfo with another cute kitty"?

If not, that's fine too. I can add it after merging. Thanks.

Reply:
Sure! Will do that ASAP